package.json

```diff
@@ -74,5 +78,16 @@
       "pre-commit": "npm test",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
+  "release": {
+    "plugins": [
+      "@semantic-release/commit-analyzer",
+      "@semantic-release/release-notes-generator",
+      "@semantic-release/changelog",
+      "@semantic-release/git",
+      "@semantic-release/npm",
```

Comment:
You need to make sure that `npm` is before `git` https://github.com/semantic-release/git#examples

Reply:
updated 8162626